content/en/real_user_monitoring/guide/proxy-rum-data.md

```diff
@@ -74,7 +74,9 @@ To successfully proxy request to Datadog:
 2. Forward the request to the URL set in the `ddforward` query parameter using the POST method.
 4. The request body must remain unchanged.
 
-Ensure the `ddforward` attribute points to a valid Datadog endpoint for your [Datadog site][2]. Failure to do so may result in an insecure configuration. 
+<div class="alert alert-error">
```

Comment:
```suggestion
<div class="alert alert-danger">
```

Reply:
Apologies, just realized that I'm the only one reviewing this. As such, I'll go ahead and commit the suggestion, as from the screenshot you provided it does appear that the intention is to have the `div` appear in red. Let me know if this is not expected, and I can make any adjustments as necessary.